### PR TITLE
 Delete redundant stress marks from 9 entries 

### PR DESCRIPTION
**Pull request by shenyileirob**
10 hits for regex `(?<![аеиоуыэюя])́`.
- 9 entries corrected;
- 1 entry pending: **шё́лк-сыре́ц**.

### CODE REVIEW

**Comment by shenyileirob** [comment]
Unfortunately I have only a scanned 1980 edition at hand, and am unable to check if the first stress mark in **шё́лк-сыре́ц** is in the original text of the 2010 edition. Would you help me with that?

**Comment by shenyileirob** [comment]
A regex search outside `/dictionary/Нарицательные` found more hits.

34 hits for regex `(?<![аеиоуыэюя])́`:
- 32 corrected;
- 2 pending: **шё́лк-сыре́ц**, **Жи́ль Бл́аз**.

**Comment by bzaar** [comment]
> Unfortunately I have only a scanned 1980 edition at hand, and am unable to check if the first stress mark in **шё́лк-сыре́ц** is in the original text of the 2010 edition. Would you help me with that?

I don't have the 2010 edition at hand either. Raised #9 so to remember to check later.

File: dictionary/Нарицательные/А.txt [hunk 1]
```diff
@@ -8855,7 +8855,7 @@
 вчера́ н; с 0
 позавчера́ н
 шу́шера ж 1a (_хлам; сброд_); мо-жо 1a (_ничтожный человек_)
-мещера́ ж 1b— [//__мещё́ра__ ж 1a] (_древнее племя, собир._)
+мещера́ ж 1b— [//__меще́ра__ ж 1a] (_древнее племя, собир._)
```
Comment (bzaar) [inline dictionary/Нарицательные/А.txt:8858]: I believe this should be a plain ё without a stress mark: https://w.wiki/44eY

File: dictionary/Нарицательные/Д.txt [hunk 2]
```diff
@@ -617,11 +617,11 @@
 макропо́д мо 1a
 испо́д м 1a
 эпо́д м 1a
-род м 1c, П2(в) (_первобытная общественная организация; ряд поколений_) ́✧ без ро́ду, без пле́мени; ни ро́ду, ни пле́мени; сто́лько-то ле́т о́т роду; э́то у него́ в роду́; на роду́ напи́сано; м 1e (_единица классификации; грамм. категория_); м 1c①: ро́д во́йск, ро́д ору́жия
+род м 1c, П2(в) (_первобытная общественная организация; ряд поколений_) ✧ без ро́ду, без пле́мени; ни ро́ду, ни пле́мени; сто́лько-то ле́т о́т роду; э́то у него́ в роду́; на роду́ напи́сано; м 1e (_единица классификации; грамм. категория_); м 1c①: ро́д во́йск, ро́д ору́жия
```
Comment (bzaar) [inline dictionary/Нарицательные/Д.txt:620]: A rogue acute accent next to ✧ (just commenting).

**Review by bzaar — CHANGES_REQUESTED** [review]
Would you mind fixing мещёра?

**Comment by bzaar** [comment]
Never mind, I'll fix мещёра myself.

**Comment by bzaar** [comment]
> Never mind, I'll fix мещёра myself.

Done: https://github.com/gramdict/zalizniak-2010/commit/280e2505f793492bc1a7e177f0190ab58851b1d1

**Comment by bzaar** [comment]
Thanks for your help!

File: dictionary/Нарицательные/А.txt (hunk 1 above)
Comment (shenyileirob) [reply dictionary/Нарицательные/А.txt:8858]: Are we trying to merely fix the digitization errors or to revise Zaliznjak (2010) in passing when appropriate?

Comment (bzaar) [reply dictionary/Нарицательные/А.txt:8858]: Good point. I think that we should keep these two goals separate. I suggest that in this repo we simply fix digitization errors. Any revisions and updates should go into a separate file/repo.

Raised #10.

**Comment by bzaar** [comment]
> * 2 pending: **шё́лк-сыре́ц**, **Жи́ль Бл́аз**.

Sorry it took me a while a get around to checking this. Please see #9 for more details.

**Pull request merged**